mne/time_frequency/tfr.py

```diff
@@ -3196,7 +3275,16 @@ def average(self, method="mean", *, dim="epochs", copy=False):
         See discussion here:
 
         https://github.com/scipy/scipy/pull/12676#issuecomment-783370228
+
+        Averaging is not supported for data containing a taper dimension.
         """
+        if "taper" in self._dims:
+            raise NotImplementedError(
+                "Averaging multitaper tapers across epochs, frequencies, or times is "
+                "not supported. If averaging across epochs, consider averaging the "
+                "epochs before computing the complex/phase spectrum."
+            )
```

Comment:
In terms of averaging for data with tapers, I went for the same approach we're using for `Spectrum` and just disallowing this.